pkg/Microsoft.Private.PackageBaseline/baseline.packages.props

```diff
@@ -323,7 +323,7 @@
       <Version>4.0.0</Version>
     </BaseLinePackage>
     <BaseLinePackage Include="System.Security.Cryptography.X509Certificates">
-      <Version>4.1.0</Version>
+      <Version>4.1.1</Version>
```

Comment:
@weshaggard When you and Eric come to an agreement I'll do whatever the consensus is :) (Maybe you'll agree with content on screen instead of a passing-in-the-hallway question).

My thought here is that any package which gets serviced next should want to say that the minver has bumped.  If that's an anti-goal I'm happy to learn that.